src/main/java/by/skopinau/transactional/outbox/demo/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package by.skopinau.transactional.outbox.demo.service.impl;
+
+import by.skopinau.transactional.outbox.demo.entity.CustomerOrder;
+import by.skopinau.transactional.outbox.demo.entity.Outbox;
+import by.skopinau.transactional.outbox.demo.repository.OrderRepository;
+import by.skopinau.transactional.outbox.demo.repository.OutboxRepository;
+import by.skopinau.transactional.outbox.demo.service.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Autowired
+    private OrderRepository orderRepository;
+    @Autowired
+    private OutboxRepository outboxRepository;
```

Comment:
Use `final` and lombok.@RequiredArgConstruct

Reply:
done